src/DynamoCoreWpf/UI/GuidedTour/GuidesValidationMethods.cs

```diff
@@ -152,41 +157,122 @@ internal static void ExecutePackageSearch(Step stepInfo, StepUIAutomation uiAuto
             if(enableFunction)
             {
                 //We need to check if the PackageManager search is already open if that is the case we don't need to open it again
-                if (ownedWindow == null)
-                {
+                if (ownedWindow != null) return;
                     stepInfo.DynamoViewModelStep.ShowPackageManagerSearch(null);
-                    PackageManagerSearchView packageManager = Guide.FindWindowOwned(stepInfo.HostPopupInfo.WindowName, stepInfo.MainWindow as Window) as PackageManagerSearchView;
-                    if (packageManager == null)
-                        return;
-                    PackageManagerSearchViewModel packageManagerViewModel = packageManager.DataContext as PackageManagerSearchViewModel;
-                    if (packageManagerViewModel == null)
-                        return;
-                    //Put the name of the Package to be searched in the SearchTextBox
-                    packageManagerViewModel.SearchText = Res.AutodeskSamplePackage;
-                    //Execute the Search
-                    packageManagerViewModel.RefreshAndSearchAsync();
-                }             
+
+                PackageManagerSearchView packageManager = Guide.FindWindowOwned(stepInfo.HostPopupInfo.WindowName, stepInfo.MainWindow as Window) as PackageManagerSearchView;
+                if (packageManager == null)
+                    return;
+                PackageManagerSearchViewModel packageManagerViewModel = packageManager.DataContext as PackageManagerSearchViewModel;
+                if (packageManagerViewModel == null)
+                    return;
+
+                //Due that we need to search the Autodesk Sample package after the inicial search is completed 
```

Comment:
typo `initial`

Reply:
@QilongTang fixed in the next commit:
https://github.com/DynamoDS/Dynamo/pull/12277/commits/7b9f128012709949425ccbf45f7bf3f52d19c89a